soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -64,6 +64,16 @@ config SPI_MCUX_LPSPI
 	default y if HAS_MCUX_LPSPI
 	depends on SPI
 
+if CODE_SEMC
+
+config FLASH_SIZE
+	default $(dt_node_reg_size_int,/soc/spi@402a8000,1,K)
+
+config FLASH_BASE_ADDRESS
+	default $(dt_node_reg_addr_hex,/soc/spi@402a8000,1)
```

Comment:
These nodes and addresses don't look right for SEMC memory. This should be:
```
config FLASH_SIZE
        default $(dt_node_reg_size_int,/memory@80000000,0,K)

config FLASH_BASE_ADDRESS
        default $(dt_node_reg_addr_hex,/memory@80000000)
```

Reply:
Thanks for your review.
Although flash address and size are not used when running in extern sram, in this case flash is indeed better when it is defined on sram. I have fixed.